Source/Atomic/UI/UI.cpp

```diff
@@ -741,6 +742,14 @@ UIWidget* UI::WrapWidget(tb::TBWidget* widget)
         return select;
     }
 
+    if (widget->IsOfType<TBPulldownMenu>())
+    {
+        UIPulldownMenu* select = new UIPulldownMenu(context_, false);
+        select->SetWidget(widget);
+        WrapWidget(select, widget);
+        return select;
+    }
```

Comment:
I wonder if we can automate this at some point, Atomic UI magic in the meantime 🐇 🎩